flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/consumer/SingleInputGate.java

```diff
@@ -616,6 +625,11 @@ public void sendTaskEvent(TaskEvent event) throws IOException {
 		}
 	}
 
+	@Override
+	public void resumeConsumption(int channelIndex) {
+		channelIndexToInputChannel[channelIndex].resumeConsumption();
```

Comment:
nit: better to assert `channelIndexToInputChannel[channelIndex] != null` in advance and give some rich informations if exception.